specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-10-15-preview/EventGrid.json

```diff
@@ -0,0 +1,8286 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-15-preview",
+    "title": "EventGridManagementClient",
+    "description": "Azure EventGrid Management Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/domains/{domainName}": {
+      "get": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Get a domain.",
+        "description": "Get properties of a domain.",
+        "operationId": "Domains_Get",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of the domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
```

Comment:
I saw in your previous swagger, the default response is defined as below.  Could you make the behavior consistent?
          "default": {
            "description": "Eventhub error response describing why the operation failed.",
            "schema": {
              "$ref": "../../../common/v1/definitions.json#/definitions/ErrorResponse"
            }
          }

Reply:
Same as above. This is not new resource/change. This already exists as part of previous swagger and not part of this PR.

PLease refere to: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json#L60

Also, this is auto-generated for us and it is consistent in all other resources in our sawagger. Hence, Won't fix. thanks.